starboard/build/toolchain/cobalt_toolchains.gni

```diff
@@ -21,6 +21,7 @@ template("cobalt_clang_toolchain") {
                            [
                              "native_linker_path",
                              "executable_extension",
+                             "tail_lib_dependencies",
```

Comment:
How about defining a `cobalt_tail_lib_dependencies`? Typically the current tail_lib_dependencies are used by the Starboard toolchain.

Reply:
I don't really mind doing this in the intermediate template (other than it would make the code a little messier), but I don't think we should do this for the gcc_toolchain/clang_toolchain templates as I don't think those should care if it's cobalt level or not